packages/main/src/Slider.ts

```diff
@@ -245,6 +265,51 @@ class Slider extends SliderBase implements IFormInputElement {
 		this._valueOnInteractionStart = undefined;
 	}
 
+	_updateValueFromInput(e: Event) {
+		const input = e.target as HTMLInputElement;
+		const value = parseFloat(input.value);
+		this._isInputValueValid = value >= this._effectiveMin && value <= this._effectiveMax;
+
+		if (!this._isInputValueValid) {
+			return;
+		}
+
+		this.value = value;
+	}
+
+	_onInputFocusOut(e: FocusEvent) {
+		const tooltipInput = e.target as Input;
+
+		this._tooltipVisibility = SliderBase.TOOLTIP_VISIBILITY.HIDDEN;
+		this._updateValueFromInput(e);
+		this._updateInputValue();
+
+		this._isInputValueValid = parseFloat(tooltipInput.value) >= this.min && parseFloat(tooltipInput.value) <= this.max;
+
+		if (!this._isInputValueValid) {
+			tooltipInput.value = this._lastValidInputValue;
```

Comment:
is it possible to keep these in a state instead of calling custom setters?

Reply:
I managed to do it half way in `the ui5-slider` but the `ui5-input`'s value update on focusout alongside "Negativ" value state behaves strange, so I had to keep one instance of a setter call in the component. 

I did not succeed to do it in Range Slider at all, I spent a lot of time and effort to do it and I'm not sure that it is possible at all with the current implementation of the sliders and the input, so I left the property settings in the code.